tests/unit/test_context.py

```diff
@@ -67,6 +67,14 @@ def test_context_credentials_and_project_can_be_set_explicitly():
     assert default_mock.call_count == 0
 
 
+def test_context_set_default_variable():
```

Comment:
style nit: maybe add an empty line after each "assert" to separate the test cases clearly?

Reply:
Not a bad idea. Done!